### PR TITLE
Refactor/utilities

### PR DESCRIPTION
**Pull request by jasmyn2244**
### What (if any) features are you implementing?
- none
### What (if anything) did you refactor?
- refactored out api calls
  - moved our data cleaning functions to utilities file
  - refactored our cleaning function to follow SRP
### Were there any issues that arose?
- n/a
### Is there anything that you need from your teammate?
- n/a
### Refactor checklist
- [x] Check variable and parameter names: are they descriptive and unique?
- [x] Is the code DRY? SRP?
- [x] Is it easy for others to read your code? (explanatory comments needed?)
- [x] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [x] Are commit messages formatted correctly?

### CODE REVIEW

File: src/utilities.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        budget: formatMoney(movie.budget),
+        revenue: formatMoney(movie.revenue),
+        runtime: `${movie.runtime} minutes`,
+        tagline: movie.tagline,
+        average_rating: movie.average_rating.toFixed(1),
+    }
+    return cleanedMovie;
+}
+
+const formatMoney = (numToConvert) => {
```
Comment (anewb87) [inline src/utilities.js:33]: Love this. Super dynamic, can use for both numbers we need to format.

Comment (Gabby-Recny) [inline src/utilities.js:33]: Much cleaner! Good work team!

**Review by Gabby-Recny — APPROVED** [review]

File: src/utilities.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+const cleanAllMovies = ({ movies }) => {
```
Comment (anewb87) [inline src/utilities.js:1]: Eyeing that 4 with this utilities.js file!

**Review by anewb87 — APPROVED** [review]
Added utilities files!

**Pull request merged**